watertap/unit_models/zero_order/CANDOP_zo.py

```diff
@@ -0,0 +1,83 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains a zero-order representation of a CANDO+P reactor unit.
+"""
+
+from pyomo.environ import Var, units as pyunits
+from idaes.core import declare_process_block_class
+from watertap.core import build_sido_reactive, constant_intensity, ZeroOrderBaseData
+
+# Some more information about this module
+__author__ = "Travis Arnold"
+
+
+@declare_process_block_class("CANDOPZO")
+class CANDOPData(ZeroOrderBaseData):
+    """
+    Zero-Order model for a CANDO+P reactor unit.
+    """
+
+    CONFIG = ZeroOrderBaseData.CONFIG()
+
+    def build(self):
+        super().build()
+
+        self._tech_type = "CANDO_P"
+
+        build_sido_reactive(self)
+
+        # Create electricity variable
+        self.electricity = Var(self.flowsheet().time,
+                               units=pyunits.kW,
+                               bounds=(0, None),
+                               doc="Electricity consumption of unit")
+        self._perf_var_dict["Electricity Demand"] = self.electricity
```

Comment:
Probably also add the unit to the dictionary:
`self._perf_var_dict["Electricity Demand (kW)"] = self.electricity`